frappe/build.py

```diff
@@ -60,7 +61,8 @@ def watch(no_compress):
 
 	# 	time.sleep(3)
 
-def make_asset_dirs(make_copy=False):
+def make_asset_dirs(make_copy=False, make_copy_force=False):
```

Comment:
`bench build --reset` instead of `--make-copy-force` ?

Reply:
Hmm. Okay. Will do the needful.

Reply:
Can't use `reset`. There's a flag already being used. Using `--restore` instead.

Reply:
Error Trace - [Travis Build](https://travis-ci.org/frappe/frappe/builds/279824587?utm_source=github_status&utm_medium=notification)